cmd/start.go

```diff
@@ -78,23 +79,24 @@ func start(cmd *cobra.Command) error {
 		return fmt.Errorf("cannot initialize network: [%v]", err)
 	}
 
+	btcChain, err := electrum.Connect(ctx, clientConfig.Bitcoin.Electrum)
+	if err != nil {
+		return fmt.Errorf("could not connect to Electrum chain: [%v]", err)
+	}
+
 	clientInfoRegistry := initializeClientInfo(
 		ctx,
 		clientConfig,
 		netProvider,
 		signing,
 		blockCounter,
+		btcChain,
 	)
 
 	// Initialize beacon and tbtc only for non-bootstrap nodes.
 	// Skip initialization for bootstrap nodes as they are only used for network
 	// discovery.
 	if !isBootstrap() {
-		btcChain, err := electrum.Connect(ctx, clientConfig.Bitcoin.Electrum)
-		if err != nil {
-			return fmt.Errorf("could not connect to Electrum chain: [%v]", err)
-		}
```

Comment:
Why require a bootstrap node to have a working connection to Bitcoin?

Reply:
I had a bad config file and it mislead me because btc metrics and diagnostics worked only if I made the btc connection accessible for all the nodes. It took me a while to figure this out because I also was skeptical about the following solution. Fixed it in [191f3d9](https://github.com/keep-network/keep-core/pull/3648/commits/191f3d91923663e8a2fe4407f8987664308f3d5e)